src/games/strategy/engine/framework/map/download/DownloadFileParser.java

```diff
@@ -26,6 +26,7 @@ private DownloadFileParser() {}
   }
 
   public static List<DownloadFileDescription> parse(final InputStream is) {
+    @SuppressWarnings("unchecked")
```

Comment:
I added the @SuppressWarnings("unchecked") if there was no easy solution fixing the problem. It's simply not possible to check a generic type. Thats why i suppressed the warning - it's recommended to do so.